Formula/openblas.rb

```diff
@@ -33,8 +32,11 @@ def install
       Hardware.oldest_cpu.upcase.to_s
     end
 
+    # make looks for `gcc-ranlib`, so we have to point it to the right one
+    ranlib = "RANLIB=gcc-ranlib-#{Formula["gcc"].version.major}"
+
     # Must call in two steps
-    system "make", "CC=#{ENV.cc}", "FC=gfortran", "libs", "netlib", "shared"
+    system "make", "CC=#{ENV.cc}", "FC=gfortran", ranlib, "libs", "netlib", "shared"
```

Comment:
How about `RANLIB=ranlib`?

Reply:
Sure, can do. Just thought this was more compact.

Reply:
No, I mean using the system `ranlib`.

Reply:
Oh, I see. Ok, let's try that.